meta-mender-raspberrypi/README.md

```diff
@@ -28,10 +28,13 @@ in addition to `meta-mender` dependencies.
 
         RPI_USE_U_BOOT = "1"
 
-        # These are simply to align with how the "stock" RPi machines are
-        # configured.
-        MENDER_PARTITION_ALIGNMENT = "4194304"
-        MENDER_BOOT_PART_SIZE_MB = "40"
+        # Having the serial terminal enabled is useful.
+        ENABLE_UART = "1"
+
+        # The boot strategy is u-boot on an sd card. We therefore
+        # disable grub and UEFI image generation.
+        MENDER_FEATURES_ENABLE_append = " mender-uboot mender-image-sd"
+        MENDER_FEATURES_DISABLE_append = " mender-grub mender-image-uefi"
```

Comment:
This is not necessary, the layer [does it for you](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-raspberrypi/conf/layer.conf#L19-L21) (indirectly).